docs/enhancements/installation-progress-bar.md

```diff
@@ -0,0 +1,116 @@
+# Cluster installation progress bar
+
+## Goals
+
+* Progress should be computed in the BE by design and not in the UI
+  * that will also make it usable from other APIs as kube-api
+* Failing workers or OLM-operators shouldn't fail the cluster installation
+  * installed clusters with failing hosts or failing OLM-operators should still reach 100% progress.
+* We should align our design to other installation processes like OCM, ACM, etc.
+  * https://marvelapp.com/prototype/gfa4i5b/screen/80132841
```

Comment:
This is related to the UI, I would suggest it's not relevant here?

Reply:
Removed.